server/neptune/workflows/minimum_revision.go

```diff
@@ -0,0 +1,12 @@
+package workflows
+
+import (
+	revisionsetter "github.com/runatlantis/atlantis/server/neptune/workflows/internal/revision_setter"
+	"go.temporal.io/sdk/workflow"
+)
+
+type SetPRMinimumRevisionRequest = revisionsetter.Request
+
+func SetMinimumRevisionForRoot(ctx workflow.Context, request SetPRMinimumRevisionRequest) error {
+	return revisionsetter.Workflow(ctx, request)
+}
```

Comment:
Pretty verbose, let's call it `PRRevision` like we're not calling our deploy function ExecuteDeployForRevision right? let's make things easier for ourselves.